server/app/com/xsn/explorer/services/XSNService.scala

```diff
@@ -26,7 +26,7 @@ trait XSNService {
 
   def getTransaction(txid: TransactionId): FutureApplicationResult[rpc.Transaction[rpc.TransactionVIN]]
 
-  def getRawTransaction(txid: TransactionId): FutureApplicationResult[JsValue]
+  def getRawTransaction(txid: TransactionId, option: Int): FutureApplicationResult[JsValue]
```

Comment:
This is not acceptable, use a proper type like a custom ADT, or, a boolean with a proper name to understand what's this for, like `hex: Boolean`.

In any case, do we still need this change? because I think we don't

Reply:
we don´t need it any more